index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="My landing page made for Coderscamp project">
+  <meta name="keywords" content="web design, html, css">
+  <meta name="author" content="Konrad Kalita">
+  <title>Landing Page</title>
+  <link rel="stylesheet" type="text/css" href="./css/styles.css">
+  <link rel="stylesheet" type="text/css" href="./css/buttons.css">
+  <link rel="stylesheet" type="text/css" href="./css/responsiveness.css">
+  <style>
+    @import url('https://fonts.googleapis.com/css?family=Raleway:400,700&display=swap');
+  </style>
+  <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+  </head>
+  <body>
+    <header>
+      <nav class="container">
+        <div class="brand">
+          <img src="./img/logo_svg.png" alt="logo" height="26px" width="26px">
+          <h1 style="padding-left: 5px;">Dev</h1>
```

Comment:
Starałbym się używać jak najmniej `inline-style` tj. styli jako atrybuty jeśli nie jest to naprawdę konieczne. Style inline są wyżej w hierarchii przez co nadpisują style importowanie z plików `css`